common/utils.ts

```diff
@@ -93,7 +94,7 @@ export const errorLoggingIssue = (() => {
 			return undefined;
 		}
 	} catch (e) {
-		console.error(e);
+		//console.error(e);
```

Comment:
this is console.error, it should either live here or be removed

Reply:
Thanks, I removed this because it was noisy when testing. It's been re-enabled.